src/test/scala/NumberPosTests.scala

```diff
@@ -0,0 +1,54 @@
+
+import org.scalatest.{FlatSpec, Matchers}
+
+class NumberPosTests extends FlatSpec with Matchers {
+
+  val num = new NumberToLongShort
+
+  "8761234563486534711" should "output correct english result" in {
```

Comment:
Test names for all these tests should be like `"For an Input of 8761234563486534711" should " output the correct result in English" in {`